packages/cm6-graphql/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "cm6-graphql",
+  "version": "0.0.0-alpha.0",
+  "description": "GraphQL language support for CodeMirror",
+  "scripts": {
+    "prepare": "cm-buildhelper src/index.ts"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "exports": {
+    "import": "./dist/index.js",
+    "require": "./dist/index.cjs"
+  },
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "devDependencies": {
+    "@codemirror/autocomplete": "6.0.0",
+    "@codemirror/buildhelper": "^0.1.16",
+    "@codemirror/language": "6.0.0",
+    "@codemirror/lint": "6.0.0",
+    "@codemirror/state": "6.1.0",
+    "@codemirror/view": "6.1.2",
+    "@lezer/common": "^1.0.0",
+    "@lezer/generator": "^1.1.0",
+    "@lezer/highlight": "^1.0.0",
+    "@lezer/lr": "^1.1.0",
+    "esbuild": "^0.14.47",
+    "graphql": "^16.5.0",
+    "graphql-language-service-interface": "2.8.4",
```

Comment:
@imolorhe these are retired and outdated, we now export all of these from `graphql-language-service`.